review/elm.json

```diff
@@ -1,39 +1,41 @@
 {
-  "type": "application",
```

Comment:
Not strictly part of this refactor but - Needed to update `elm-review` to run lint